src/SFA.DAS.FAT.Web/Views/Courses/CourseProviderDetail.cshtml

```diff
@@ -4,9 +4,14 @@
 @using SFA.DAS.FAT.Web.Extensions
 @using DeliveryModeType = SFA.DAS.FAT.Web.Models.DeliveryModeType
 
+@section feedbackstyles {
+    <link rel="stylesheet" media="screen" href="https://www.gov.uk/assets/government-frontend/application-0e6f1b796b951ca0e07ce65f537ba3045e163b208c897f53b1fd4f36993385cc.css"/>
```

Comment:
You probably want to download these styles locally and reference them as part of the project. There isn't a guarantee that will stay live nor that they won't change. I can see that it's a front end assets site but I'm not familiar with it being used by AS, I'd ask James King, but aware he's on holiday.